internal/util/operator-registry/package_manifest.go

```diff
@@ -18,19 +18,18 @@ import (
 	"fmt"
 
 	"github.com/operator-framework/operator-registry/pkg/registry"
-	"github.com/pkg/errors"
 )
 
 func ValidatePackageManifest(pkg *registry.PackageManifest) error {
 	if pkg.PackageName == "" {
-		return errors.New("package name cannot be empty")
+		return fmt.Errorf("package name cannot be empty")
```

Comment:
No need to change these from `errors.New` to `fmt.Errorf`. Instead, use the std `errors` package over `github.com/pkg/errors`.

Reply:
One of the items mentioned in #2308 was about using native go lang errors. Hence I changed this. Please confirm if dependency on `github.com/pkg/errors` is fine to have.
I thought its better to remove this dependency.

Reply:
You can use the `”errors"` package, which is in the standard library.

Reply:
Done.